packages/js/client/src/PolywrapClient.ts

```diff
@@ -1,11 +1,5 @@
 import { InvokerOptions, TryResolveUriOptions } from "./types";
-import {
-  PolywrapClientConfig,
-  PolywrapCoreClientConfig,
-  buildPolywrapCoreClientConfig,
-  sanitizeUri,
-  ClientConfig,
-} from "./legacy";
+import { sanitizeUri } from "./legacy";
```

Comment:
Can we delete the legacy config types and rename the folder to something like "helpers" or "util"? Or are we not ready for that yet?

Reply:
okay, i'm just now reviewing your other PR and I see this is done there